tests/test_models.py

```diff
@@ -75,3 +75,33 @@ def test_num_classes():
         # should raise error:
         xrv.models.DenseNet(weights="all", num_classes=4)
         
+def test_normalization_check():
+    
+    models = [xrv.models.DenseNet(weights="densenet121-res224-all"),
+             xrv.models.ResNet(weights="resnet50-res512-all")]
+    
+    incorrect_ranges = [
+        [0, 1],
+        [-1, 1],
+        [0, 1024],
```

Comment:
I think it would be more likely that someone made a mistake in normalization than an image with no negative values if it was supposed to be [-1024, 1024]. I think it would be better to show a warning and have them verify it is correct than to silently fail.